python/lsst/ip/isr/ptcDataset.py

```diff
@@ -561,14 +589,21 @@ def fromTable(cls, tableList):
                     inDict['ptcTurnoff'][ampName] = np.nan
             else:
                 inDict['ptcTurnoff'][ampName] = record['PTC_TURNOFF']
-            if calibVersion == 1.1:
+            if calibVersion < 1.2:
                 inDict['histVars'][ampName] = np.array([np.nan])
                 inDict['histChi2Dofs'][ampName] = np.array([np.nan])
                 inDict['kspValues'][ampName] = np.array([0.0])
             else:
                 inDict['histVars'][ampName] = record['HIST_VARS']
                 inDict['histChi2Dofs'][ampName] = record['HIST_CHI2_DOFS']
                 inDict['kspValues'][ampName] = record['KS_PVALUES']
+            if calibVersion >= 1.3:
+                inDict['noiseMatrix'][ampName] = record['NOISE_MATRIX']
+                inDict['noiseMatrixNoB'][ampName] = record['NOISE_MATRIX_NO_B']
+            else:
+                nanMatrix = np.full_like(inDict['aMatrix'][ampName], np.nan)
+                inDict['noiseMatrix'][ampName] = nanMatrix
+                inDict['noiseMatrixNoB'][ampName] = nanMatrix
```

Comment:
I know this is a small thing, but I think that it would be better if we used the same construction for both calibVersion checks.  Something like this:
```
            if calibVersion < 1.2:
                inDict['histVars'][ampName] = np.array([np.nan])
                inDict['histChi2Dofs'][ampName] = np.array([np.nan])
                inDict['kspValues'][ampName] = np.array([0.0])
            else:
                inDict['histVars'][ampName] = record['HIST_VARS']
                inDict['histChi2Dofs'][ampName] = record['HIST_CHI2_DOFS']
                inDict['kspValues'][ampName] = record['KS_PVALUES']
            if calibVersion < 1.3:
                nanMatrix = np.full_like(inDict['aMatrix'][ampName], np.nan)
                inDict['noiseMatrix'][ampName] = nanMatrix
                inDict['noiseMatrixNoB'][ampName] = nanMatrix
                inDict['noiseMatrix'][ampName] = record['NOISE_MATRIX']
                inDict['noiseMatrixNoB'][ampName] = record['NOISE_MATRIX_NO_B']
            else:
                inDict['noiseMatrix'][ampName] = record['NOISE_MATRIX']
                inDict['noiseMatrixNoB'][ampName] = record['NOISE_MATRIX_NO_B']
```

Reply:
Sounds good, I changed it to this. Thanks.